### DISCUSSION TITLE
Error discovery in Log Sources - logflare.app

### DISCUSSION THREAD

**Discussion by vulrun** [supabase/supabase#21311] [General]
I need to bring to your attention an issue I've encountered while exploring logs from our sources. Since this morning, I've been experiencing an unexpected error, which is hindering my ability to access and analyze the necessary data.

Given the importance of this task and the impact it may have on our operations, I kindly request that you look into this matter at your earliest convenience. Swift resolution is crucial to ensure that we can continue our work effectively and without interruption. Attaching a screenshot for your reference.

![Screenshot from 2024-02-14 21-21-36](https://github.com/supabase/supabase/assets/20625306/cd8bffa9-864f-4e1e-a123-c82aee934590)

**Comment by Ziinc** [comment]
Hi if this issue is still persisting, please open a ticket so that I can investigate this for you.
https://supabase.com/support

**Reply by vulrun** [reply]
@Ziinc Issue has been resolved, you can closed the ticket.
